packages/web/src/pages/chat-page/components/ChatMessageListItem.tsx

```diff
@@ -85,6 +97,15 @@ export const ChatMessageListItem = (props: ChatMessageListItemProps) => {
       })}
     >
       <div className={styles.bubble}>
+        {links
+          .filter((link) => link.type === 'url' && link.isLink)
+          .slice(0, 1)
```

Comment:
just curious why slice rather than `[0]`

Reply:
originally had coded it for multiple previews, was lazy when I changed it to one 😅 

`[0]` probably makes more sense...